docs/tools/firmware.sh

```diff
@@ -832,11 +845,11 @@ case "$os" in
 				echo "Copying this script to EFI"
 				cp "$0" "/Volumes/${EFILABEL}/firmware.sh" 2>/dev/null \
 					|| curl -s https://wiki.t2linux.org/tools/firmware.sh > "/Volumes/${EFILABEL}/firmware.sh" \
-					|| (echo && cat <<- EOF && read -r
+					|| (
+						echo && cat <<- EOF && [ "$interactive" = "true"] && echo "Press enter after you have copied" && read -r
```

Comment:
Shouldn't there be a blank line before Press enter after you have copied?

Reply:
It looks exactly the same as before, but I can add a extra newline if you want.

Reply:
> It looks exactly the same as before, but I can add a extra newline if you want.

It's fine then